SELECTOR_INFO.md

```diff
@@ -2591,6 +2598,30 @@ Description: Geolocation prompt container
 Location: Address bar
 Path to .json: modules/data/navigation.components.json
 ```
+```
+Selector Name: permission-popup-menulist
```

Comment:
I feel like the name should reflect the "blocked" status, so something like `permission-popup-audio-video-blocked`

Reply:
Done.